docs/Networking.asciidoc

```diff
@@ -258,7 +262,7 @@ zypper in libcap-progs
 setcap CAP_NET_ADMIN=ep /usr/bin/qemu-system-x86_64
 ---------------
 
-*Configure Network Manager*
+*Configure network interfaces*
```

Comment:
thank you, I just stumbled over this one yesterday! :D

Reply:
I suggested that change in a previous review comment :-)